tests/GistFixtureHelpers.php

```diff
@@ -6,4 +6,10 @@ private function loadFixture($path)
     {
         return json_decode(file_get_contents(base_path() . '/tests/fixtures/gists/' . $path), true);
     }
+
+    // Someone please help me name this
+    private function loadNonJsonFixture($path)
```

Comment:
I'd rename the other one to `loadJsonFixture` and make this one just `loadFixture`.